css/stylesheet.css

```diff
@@ -156,3 +161,43 @@ h4 {
     border: 2px solid #F8D5E2;
 }
 
+.contacts-hometop {
+    background-color: #cee4c7;
+}
+
+.contacts-social a {
+    color: #cee4c7;
+}
+
+.contacts-social a:hover {
+    color: #F8D5E2;
+}
+
+.contacts-hometop nav a:hover {
+    color: #F8D5E2;
+}
+
+.contacts {
+    width: 30%;
+    overflow: auto;
+    position: absolute;
```

Comment:
try to stay away from position absolute, much safer to go with a div with a width that has margin 0 auto, this will put it in the middle of the page

Reply:
did as you said